simulationdataschema/model_method.py

```diff
@@ -47,6 +47,6 @@ class ModelMethod(ArchiveSection):
 
     normalizer_level = 1
 
-    def normalize(self, archive, logger):
+    def normalize(self, archive, logger) -> None:
```

Comment:
Great! Don't forget the other types: `EntryArchive` and `BoundLogger`.

Reply:
There are some circular import problems when trying to get `EntryArchive`. I opted for leaving out specifying types inputs in the `normalize()`.

Reply:
I see... Then never mind.